chromium/background-scripts/update.js

```diff
@@ -170,7 +187,9 @@ async function applyStoredRulesets(rulesets_obj){
   const rulesets_jsons = await Promise.all(rulesets_promises);
   if(rulesets_jsons.join("").length > 0){
     for(let rulesets_json of rulesets_jsons){
```

Comment:
It seems to me that EFF's update channels will always have the top priority in rewriting URLs, while other channels are prioritize by insertion order. If the coverage of different channels overlaps, it might not works as the users expected, e.g. some channels might never apply. If this is the case, it might be good to have a way to prioritize the channels (possibly in later PR/ release). Please correct me if I am wrong.

Reply:
Yes I think prioritization of update channels makes sense, but this will require some re-engineering.  This would be a good task for a future PR.